src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -1380,7 +1380,12 @@ NativeParameterBinderController.ArgumentPassingStyle switch
             string rawPath =
                 context.EngineSessionState.GetNamespaceCurrentLocation(
                     context.ProviderNames.FileSystem).ProviderPath;
-            startInfo.WorkingDirectory = WildcardPattern.Unescape(rawPath);
```

Comment:
The current path is used a lot in code base, so I guess we will run into problems in some other code paths when the PowerShell's current working directory is gone. One thing for sure is the counterpart code in `Start-Process`, where we also set `WorkingDirectory`: https://github.com/PowerShell/PowerShell/blob/master/src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs#L1920

Reply:
> The current path is used a lot in code base, so I guess we will run into problems in some other code paths when the PowerShell's current working directory is gone.

So a more general question is: do we want to claim that we support the scenario where PowerShell's current working directory gets deleted?

Reply:
I don't think this is an important scenario to support.

Reply:
I agree that PS7's own current working directory getting deleted is out of scope and hasn't been brought up by users

Reply:
@SteveL-MSFT I'm a little confused. Here, the `rawPath` is the PS7's own current working directory, right?

Reply:
Discussed this with @daxian-dbw and have a better understanding of the ask.  Will fix `Start-Process` to match executing directly.  Any other edge cases will have to be reported and handled on a case-by-case basis.